docker_push_latest_if_changed.py

```diff
@@ -1,5 +1,182 @@
 #!/usr/bin/env python3
 import argparse
+import hashlib
+import logging
+from collections import namedtuple
+
+import docker
```

Comment:
From the readme:

> The tool is a single file which only has dependencies on python3 and a
> working `docker` executable.  You can simply drop the file onto your `PATH`:

Reply:
(aka: adding a dependency on dockerpy breaks the current packaging and portability, so 👎 on that)

Reply:
I should really learn how to read
Made it stop using dockerpy and just use the cli instead